tests/tt_metal/tt_metal/perf_microbenchmark/dispatch/test_prefetcher.cpp

```diff
@@ -882,7 +885,7 @@ void gen_packed_read_test(Device *device,
     bool done = false;
     while (!done) {
         uint32_t packed_read_page_size = std::rand() % 3 + 9; // 512, 1024, 2048
-        uint32_t n_sub_cmds = (std::rand() % 7) + 1;
+        uint32_t n_sub_cmds =  relay_max_packed_paged_submcds ?  35 : (std::rand() % 7) + 1;
```

Comment:
Can't we access the value from cq_commands.hpp instead of hardcoding 35?

Reply:
Good point, updated